src/styles/main.scss

```diff
@@ -0,0 +1,8 @@
+.react__portal__tooltip__card {
+  position: absolute;
+  padding: 5px;
+  background: #fff;
+  box-shadow: 0 0 8px rgba(0,0,0,.3);
+  border-radius: 3px;
+  z-index: 50;
+}
```

Comment:
newline!

Reply:
I see that you have put all the styles into one css file. Maybe we can start doing http://getbem.com/introduction/ while we are at it.

For example, there is one big block `ReactPortalTooltip` and then `ReactPortalTooltip__card`

Reply:
@tanqhnguyen  yes it's a good suggestion.